io.github.cdrummond.melodeon.yml

```diff
@@ -0,0 +1,27 @@
+id: io.github.cdrummond.melodeon
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '6.8'
+command: melodeon
+rename-desktop-file: melodeon.desktop
+rename-icon: melodeon
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --own-name=org.mpris.MediaPlayer2.melodeon
```

Comment:
is it possible to make this `org.mpris.MediaPlayer2.$flatpak-id` upstream? then this permission will not be needed.